assignments/callbacks.js

```diff
@@ -23,25 +23,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
+  // getLength passes the length of the array into the callback.	  // getLength passes the length of the array into the callback.
+  return cb(arr.length);
+}	
+ getLength (items, function(length) {
+  console.log(length);
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.pop());
```

Comment:
pop is used to remove an item from an array. compare to solution code: 
function last(arr, cb) {
  cb(arr[arr.length - 1]);
}